ubatch/data_request.py

```diff
@@ -21,7 +21,7 @@ class DataRequestBufferFull(Exception):
 
 
 class DataRequest(Generic[T, S]):
-    def __init__(self, data: T, timeout: float):
+    def __init__(self, data: T = None, timeout: float = 1, args: tuple = (), kwargs: dict = {}):
```

Comment:
**data** is not longer in use, but is still in the code. Why not remove data o maybe use data as a dict to store args,kwargs?

Example:

```
data = {
    "args": []
    "kwargs": {"foo": "bar",}
}
```